### ISSUE TITLE
Error where it should not be

### ISSUE THREAD

**Issue by GoodGuyNick** [eslint/eslint#8355]
**Tell us about your environment**

* **ESLint Version:** 3.18.0
* **Node Version:** 6.10.1
* **npm Version:** 3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?**
none(default)?

**Please show your full configuration:**

```
{
  "plugins": ["html"],
  "extends": "airbnb",
  "env": {
        "browser": true,
        "node": true
  },
  "rules": {
    "react/jsx-filename-extension": ["error", { "extensions": [".js", ".jsx"] }]
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

Code from create-react-app
```js
import React, { Component } from 'react';
import logo from './logo.svg';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <div className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h2>Welcome to React</h2>
        </div>
        <p className="App-intro">
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    );
  }
}

export default App;
```

**What did you expect to happen?**
There are indentation of 10 space characters but it shows error
**What actually happened? Please include the actual, raw output from ESLint.**
14:2   error  Expected indentation of 10 space characters but found 1  react/jsx-indent

Error on line ```To get started, edit <code>src/App.js</code> and save to reload.```

├── eslint@3.18.0
├── eslint-config-airbnb@14.1.0
├── eslint-plugin-html@2.0.1
├── eslint-plugin-import@2.2.0
├── eslint-plugin-jsx-a11y@4.0.0
├── eslint-plugin-react@6.10.1

**Comment by not-an-aardvark** [comment]
It looks like the problem is coming from the `jsx-indent` rule, which is part of [`eslint-plugin-react`](https://github.com/yannickcr/eslint-plugin-react). Please report the issue there instead.
